packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts

```diff
@@ -17,6 +17,11 @@ type GridFilterItemApplier = {
   item: GridFilterItem;
 };
 
+type GridFilterItemApplierNotAggregated = (
```

Comment:
Side note, we should probably stop using "aggregated" for this filtering """aggregation"""" now that we have a feature named "aggregation"